src/test/java/filterEventsOnTheMapByCategories/US_FilterEventsOnTheMapByCategoriesWithChrome.java

```diff
@@ -113,6 +117,16 @@ public void setUpPreconditions(){
     public boolean isElementEnabled(WebElement element){
         return element.isEnabled();
     }
+    public boolean isElementClickable(WebElement element, By locator){
```

Comment:
If element clickable, we return true. But most likely among the pair (WebElement element, By locator),  
WebElement element is superfluous. I think...